Dockerfile

```diff
@@ -1,7 +1,13 @@
 FROM ubuntu:16.04
 
-RUN apt-get update && apt-get install -y curl  libunwind8 libicu55
-RUN curl -LO https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-alpha.9/powershell_6.0.0-alpha.9-1ubuntu1.16.04.1_amd64.deb
-RUN dpkg -i powershell_6.0.0-alpha.9-1ubuntu1.16.04.1_amd64.deb
+ENV POWERSHELL_VERSION="6.0.0-alpha.9"
 
-Entrypoint [ "powershell" ]
+RUN apt-get update && \
+    apt-get install -y curl libunwind8 libicu55 && \
+    curl -LO https://github.com/PowerShell/PowerShell/releases/download/v"$POWERSHELL_VERSION"/powershell_"$POWERSHELL_VERSION"-1ubuntu1.16.04.1_amd64.deb && \
+    dpkg -i powershell_"$POWERSHELL_VERSION"-1ubuntu1.16.04.1_amd64.deb && \
```

Comment:
Yup: https://raw.githubusercontent.com/PowerShell/PowerShell/master/tools/download.sh

Or it could come from source, as you said.

If we add the ability to get a particular version, we should probably do it in that script so everyone can use it.

As it is, plugging in the version here isn't reliable if the package names change or the iteration updates.

So many problems get solved once we have repository packages